llvm/lib/Target/SystemZ/SystemZISelLowering.cpp

```diff
@@ -2757,6 +2797,21 @@ static unsigned CCMaskForCondCode(ISD::CondCode CC) {
 #undef CONV
 }
 
+static unsigned CCMaskForSystemZCCVal(unsigned CC) {
```

Comment:
I don't think this is needed, this is simply `1 << (3 - CC)` or something.  You have similar code inline elsewhere anyway, so you might just as well inline this too.